Add bech32-ibc to cosmos-sdk-proto-althea

ChristianBorst opened the pull request.

jkilpatr commented on proto-build/src/main.rs:
```diff
@@ -19,14 +19,16 @@ use walkdir::WalkDir;
+
 /// The Cosmos SDK commit or tag to be cloned and used to build the proto files
-const COSMOS_SDK_REV: &str = "v0.44.1";
+const COSMOS_SDK_REV: &str = "1c4b62d70267374eb11f182ce1e97517fe7b4b56";
```
why this specific rev rather than v0.44.2?

jkilpatr commented on proto-build/src/main.rs:
```diff
@@ -147,6 +153,10 @@ fn update_submodules() {
 
+    info!("Updating osmosis-labs/bech32ibc submodule...");
+    run_git(&["-C", BECH32IBC_DIR, "fetch"]);
+    run_git(&["-C", BECH32IBC_DIR, "reset", "--hard", BECH32IBC_REV]);
```
I preffered using libgit2 to submodules where you actually had rust bindings to git. But they ripped it out when 'upstreamed' oh well just griping.

ChristianBorst replied:
:angry:

jkilpatr reviewed (approved):
Needs a rebase and a few comments but looks good, increment the version and push to cargo

ChristianBorst committed:
Add osmosis-labs/bech32-ibc protos

Unfortunately osmosis didn't add a third_party folder to their
bech32-ibc repo, so I needed to fork it at althea-net/bech32-ibc and add
that as this repo expects that any project have all its proto dependencies
in third_party. I also overwrote the v0.2.0-rc2 tag to be the commit with
that issue resolved.
